src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2427,6 +2428,7 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             IqsmsTransportFactory::class => 'notifier.transport_factory.iqsms',
             LightSmsTransportFactory::class => 'notifier.transport_factory.lightsms',
             LinkedInTransportFactory::class => 'notifier.transport_factory.linkedin',
+            MailjetNotifierTransportFactory::class => 'notifier.transport_factory.mailjet',
```

Comment:
```suggestion
            MailjetTransportFactory::class => 'notifier.transport_factory.mailjet',
```

Reply:
There is already an MailjetTransportFactory used by Symfony Mailer component

Reply:
There is already an MailjetTransportFactory used by Symfony Mailer component